### ISSUE TITLE
losses.py

### ISSUE THREAD

**Issue by yangyt-2020** [HobbitLong/SupContrast#58]
Hi,HobbitLong,
Thank you for this study ,code and paper.
In losses.py ，Why are you doing this?

logits = anchor_dot_contrast - logits_max.detach()

Thanks!!!
